src/Extractor/ContentExtractor.php

```diff
@@ -184,31 +184,36 @@ public function process($html, $url, SiteConfig $siteConfig = null, $smartTidy =
             $this->config['src_lazy_load_attributes'][] = $this->siteConfig->src_lazy_load_attr;
         }
 
+        $this->logger->debug('Actual site config: ', ['siteConfig' => $this->siteConfig]);
+
         // do string replacements
         if (!empty($this->siteConfig->find_string)) {
             if (\count($this->siteConfig->find_string) === \count($this->siteConfig->replace_string)) {
                 $html = str_replace($this->siteConfig->find_string, $this->siteConfig->replace_string, $html, $count);
-                $this->logger->log('debug', 'Strings replaced: {count} (find_string and/or replace_string)', ['count' => $count]);
+                $this->logger->info('Strings replaced: {count} (find_string and/or replace_string)', ['count' => $count]);
             } else {
-                $this->logger->log('debug', 'Skipped string replacement - incorrect number of find-replace strings in site config');
+                $this->logger->info('Skipped string replacement - incorrect number of find-replace strings in site config');
             }
             unset($count);
         }
 
+        $this->logger->debug('HTML after site config strings replacements: ', ['html' => $html]);
+
         // load and parse html
         $parser = $this->siteConfig->parser();
 
         if (!\in_array($parser, $this->config['allowed_parsers'], true)) {
-            $this->logger->log('debug', 'HTML parser {parser} not listed, using {default_parser} instead', ['parser' => $parser, 'default_parser' => $this->config['default_parser']]);
+            $this->logger->info('HTML parser {parser} not listed, using {default_parser} instead', ['parser' => $parser, 'default_parser' => $this->config['default_parser']]);
             $parser = $this->config['default_parser'];
         }
 
-        $this->logger->log('debug', 'Attempting to parse HTML with {parser}', ['parser' => $parser]);
+        $this->logger->info('Attempting to parse HTML with {parser}', ['parser' => $parser]);
 
         $this->readability = $this->getReadability($html, $url, $parser, $this->siteConfig->tidy() && $smartTidy);
         $tidied = $this->readability->tidied;
 
-        $this->logger->log('debug', 'Body size after Readability: {length}', ['length' => \strlen($this->readability->dom->saveXML())]);
+        $this->logger->info('Body size after Readability: {length}', ['length' => \strlen($this->readability->dom->saveXML())]);
+        $this->logger->debug('Body after Readability: ', ['dom_saveXML' => $this->readability->dom->saveXML()]);
```

Comment:
```suggestion
        $this->logger->debug('Body after Readability', ['dom_saveXML' => $this->readability->dom->saveXML()]);
```

Reply:
Resolved in https://github.com/j0k3r/graby/pull/184/commits/0a7d2ca09e7a0f81d9d66ac24b4acad48990d09e